dotCMS/src/main/java/com/dotcms/publishing/PushPublishFiltersInitializer.java

```diff
@@ -55,9 +55,13 @@ public void init() {
     protected void loadFilter(final Path path){
         final String fileName = path.getFileName().toString();
         Logger.info(PushPublishFiltersInitializer.class, " ymlFileName:  " + fileName);
-        final FilterDescriptor filterDescriptor = YamlUtil.parse(path,FilterDescriptor.class);
-        filterDescriptor.setKey(fileName);
-        Logger.info(PushPublishFiltersInitializer.class, filterDescriptor.toString());
-        APILocator.getPublisherAPI().addFilterDescriptor(filterDescriptor);
+        try {
+            final FilterDescriptor filterDescriptor = YamlUtil.parse(path, FilterDescriptor.class);
+            filterDescriptor.setKey(fileName);
+            Logger.info(PushPublishFiltersInitializer.class, filterDescriptor.toString());
+            APILocator.getPublisherAPI().addFilterDescriptor(filterDescriptor);
+        }catch(Exception e) {
```

Comment:
Codacy found an issue: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5796590)